docs/genai/howto/install.md

```diff
@@ -23,61 +29,149 @@ The CUDA toolkit can be downloaded from the [CUDA Toolkit Archive](https://devel
 
 Ensure that the `CUDA_PATH` environment variable is set to the location of your CUDA installation.
 
+
 ## Python packages
 
-Note: only one of these packages should be installed in your application.
+Note: only one of these sets of packages (CPU, DirectML, CUDA) should be installed in your environment.
 
 ### CPU
 
+#### x64
+
 ```bash
-pip install numpy
+# Install ONNX Runtime nightly. Update this to the released version when it is available.
+pip install ort-nightly --extra-index-url
+https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ort-nightly/pypi/simple
+
+
+pip install onnxruntime-genai --pre
+```
+
+#### Arm64
+
+```bash
+# Install ONNX Runtime nightly QNN version (which supports Arm CPU). Update this to the released version when it is available.
+pip install ort-nightly-qnn --extra-index-url
+https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ort-nightly/pypi/simple
+
 pip install onnxruntime-genai --pre
 ```
 
-### DirectML
 
-Append `-directml` for the library that is optimized for DirectML on Windows
+### DirectML
 
 ```bash
-pip install numpy
+# Install the nightly version of ONNX Runtime. Update this to the released version when it is available.
+pip install ort-nightly-directml --extra-index-url
+https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ort-nightly/pypi/simple
```

Comment:
```suggestion
pip install -i
https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ORT-Nightly/pypi/simple/ "ort-nightly-directml~=1.19.0.dev"
```

Reply:
Reverted the Python install updates for now